### ISSUE TITLE
When using nocss import, could not find react-redux context value; please ensure the component is wrapped in a <Provider>

### ISSUE THREAD

**Issue by ohardy** [wbkd/react-flow#983]
- Error: `could not find react-redux context value; please ensure the component is wrapped in a <Provider>`

```
Call Stack
 useReduxContext
  bundle.js:278771:11
 useSelector
  bundle.js:278954:28
 MiniMap
  bundle.js:288262:24
 renderWithHooks
  bundle.js:265432:18
 updateFunctionComponent
  bundle.js:267803:20
 updateSimpleMemoComponent
  bundle.js:267662:10
 updateMemoComponent
  bundle.js:267539:14
 beginWork
  bundle.js:269582:16
 HTMLUnknownElement.callCallback
  bundle.js:254392:14
 Object.invokeGuardedCallbackDev
  bundle.js:254441:16
```

**Comment by moklick** [comment]
What are you tying to do? If you want to use one of the hooks, please wrap your app with the `ReactFlowProvider` https://reactflow.dev/docs/api/components/provider/

**Comment by zlangbert** [comment]
I hit the same issue, and after trying to come up with a smaller reproduction, figured out what went wrong for me. TLDR: don't mix imports from `react-flow-renderer` and `react-flow-renderer/nocss`.

In my case I had one file with the `<ReactFlow/>` component which I had switched to use `react-flow-renderer/nocss`, and a component in another file still using `import { useStoreState } from "react-flow-renderer"`. Once I aligned the imports it was able to find the provider again.
